scripts/neighbor_advertiser

```diff
@@ -115,6 +115,8 @@ def get_loopback_addr(ip_ver):
     loopback_addr = ''
 
     for intf in loopback_intfs.keys():
+        if not isinstance(intf, tuple):
+             continue
```

Comment:
there is readability for this code, not sure what is the intention here. Is it better to have a function to define the intention and use the function so that developer knows the intention here.

Reply:
Addressed